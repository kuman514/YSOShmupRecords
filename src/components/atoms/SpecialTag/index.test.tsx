import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { beforeEach, describe, expect, it } from 'vitest';
import 'jest-styled-components';

import { SpecialTag } from '.';

describe('SpecialTag', () => {
  beforeEach(() => {
    cleanup();
  });

  it('should have a snapshot match', () => {
    const { container } = render(<SpecialTag>kuman514</SpecialTag>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
