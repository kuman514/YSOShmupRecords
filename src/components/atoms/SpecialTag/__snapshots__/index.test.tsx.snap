// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SpecialTag > should have a snapshot match 1`] = `
.c0 {
  padding: 4px 8px;
  justify-content: center;
  align-items: center;
  background-color: var(--gray-color);
  border-radius: 4px;
  font-size: 12px;
  font-weight: 400;
  color: var(--white-color);
}

<span
  class="c0"
>
  kuman514
</span>
`;
